code/modules/mob/say.dm

```diff
@@ -2,17 +2,21 @@
 /mob/verb/say_verb(message as text)
 	set name = "Say"
 	set category = "IC"
+	var/regex/whisper_finder = regex("^\\.w\\s+|^\\.w")
 	if(GLOB.say_disabled)	//This is here to try to identify lag problems
-		to_chat(usr, "<span class='danger'>Speech is currently admin-disabled.</span>")
+		to_chat(usr, "<span class='danger'>Speech has currently been disabled by the admins.</span>")

```

Comment:
This is actually grammatically incorrect, meine Freunde.

Reply:
Or would it be "mein Freund"..?

I always get it mixed up.

Reply:
"mein Freund" is singular, "meine Freunde" is plural.

Reply:
Also, could you point out the grammatical error to me? I am failing to spot it.